### PR TITLE
change blockquotes to admonitions

### PR DESCRIPTION
**Pull request by alicewriteswrongs**
I _believe_ I 1) got them all and 2) organized them into commits correctly.

The easiest way to make sure is probably to look commit-by-commit and make sure that the changes are sensible and the same across the different versions.

Also it's worth asking if the appropriate admonition is used in every case - I went for a 'tip' or 'caution' where I thought it was warranted.

### CODE REVIEW

File: docs/config/01-overview.md [hunk 1]
```diff
@@ -38,7 +38,9 @@ If the app does not need to run on legacy browsers (IE11 and Edge 18 and below),
 In addition to creating es5 builds, apps may also be interested in enable runtime options to __support__ legacy browsers. 
 See [config extras](/config-extras) for more information.
 
-> As of Stencil v3, legacy browser support is deprecated, and will be removed in a future version major of Stencil.
+:::info
+As of Stencil v3, legacy browser support is deprecated, and will be removed in a future version major of Stencil.
```
Comment (tanner-reits) [inline docs/config/01-overview.md:42]:
```suggestion
As of Stencil v3, legacy browser support is deprecated, and will be removed in a future major version of Stencil.
```

File: versioned_docs/version-v2/introduction/upgrading-to-stencil-three.md [hunk 2]
```diff
@@ -8,7 +8,9 @@ contributors:
 
 # Upgrading to Stencil v3.0.0
 
-> Stencil 3.0.0 is pre-release software. These instructions are for users looking to try an early version of the software
+:::caution
+Stencil 3.0.0 is pre-release software. These instructions are for users looking to try an early version of the software
+:::
```
Comment (tanner-reits) [inline versioned_docs/version-v2/introduction/upgrading-to-stencil-three.md:13]: This should go away, no?

File: versioned_docs/version-v3.0/config/01-overview.md [hunk 3]
```diff
@@ -38,7 +38,9 @@ If the app does not need to run on legacy browsers (IE11 and Edge 18 and below),
 In addition to creating es5 builds, apps may also be interested in enable runtime options to __support__ legacy browsers. 
 See [config extras](/config-extras) for more information.
 
-> As of Stencil v3, legacy browser support is deprecated, and will be removed in a future version major of Stencil.
+:::info
+As of Stencil v3, legacy browser support is deprecated, and will be removed in a future version major of Stencil.
```
Comment (tanner-reits) [inline versioned_docs/version-v3.0/config/01-overview.md:42]:
```suggestion
As of Stencil v3, legacy browser support is deprecated, and will be removed in a future major version of Stencil.
```

**Review by tanner-reits — CHANGES_REQUESTED** [review]
Couple super small fixes

File: versioned_docs/version-v2/introduction/upgrading-to-stencil-three.md (hunk 2 above)
Comment (rwaskiewicz) [reply versioned_docs/version-v2/introduction/upgrading-to-stencil-three.md:13]: Ooh, yeah. Agreed - I missed that yesterday

**Comment by alicewriteswrongs** [comment]
@tanner-reits thanks for catching those, should be ready for another look

**Review by tanner-reits — APPROVED** [review]

File: docs/output-targets/custom-elements-bundle.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
+---
```
Comment (rwaskiewicz) [inline docs/output-targets/custom-elements-bundle.md:1]: @alicewriteswrongs I think this got re-added on the rebase. Can you rm it?

File: versioned_docs/version-v3.0/output-targets/custom-elements-bundle.md [hunk 5]
```diff
@@ -0,0 +1,200 @@
+---
```
Comment (rwaskiewicz) [inline versioned_docs/version-v3.0/output-targets/custom-elements-bundle.md:1]: @alicewriteswrongs I think this got re-added on the rebase. Can you rm it?

**Review by rwaskiewicz — CHANGES_REQUESTED** [review]

File: docs/output-targets/custom-elements-bundle.md (hunk 4 above)
Comment (alicewriteswrongs) [reply docs/output-targets/custom-elements-bundle.md:1]: 🤦‍♀️

**Review by rwaskiewicz — APPROVED** [review]

**Review by tanner-reits — APPROVED** [review]

**Pull request merged**